text/0061-error-format-and-definitions.md

```diff
@@ -89,6 +89,123 @@ HTTP code `400 Bad Request`
 }
 ```
 
+- The `:errorMessage` is inferred when the message is generated.
+
+---
+
+## missing_parameter
```

Comment:
Is this error tied to this particular spec?

Reply:
Yes, this is an error to distinguish a parameter that would be invalid and a mandatory parameter that is not filled in for a resource.

So we would have a more general `missing_parameter` error when a mandatory parameter is not filled in for a resource in the payload. 

We keep the `invalid_:resourceName_:parameterName` convention that we already had with `invalid_index_uid` for example when the field found is not valid.

Reply:
So, to confirm: this error is introduced in this spec, but we expect to use it for other endpoints in the future, correct?

Reply:
Yes @guimachiavelli!